### PR TITLE
Added German Language Support

### PR DESCRIPTION
**Pull request by dombyte**

### CODE REVIEW

File: exampleSite/config.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 baseURL = '/'
-languageCode = 'en'
-defaultContentLanguage = "en"
+languageCode = 'de'
+defaultContentLanguage = "de"
```
Comment (floyd-li) [inline exampleSite/config.toml:3]: please do not change the default language of the example site

File: .gitignore
Comment (floyd-li) [inline .gitignore:1]: i don't think you need to add this `.gitignore` file

**Review by floyd-li — CHANGES_REQUESTED** [review]
Thanks for your contribution! 
just little suggestions, you may need to change these then it can be merged!

File: exampleSite/config.toml (hunk 1 above)
Comment (dombyte) [reply exampleSite/config.toml:3]: Sure!

**Comment by floyd-li** [comment]
Hey dude, did you delete the `i18n/de.toml` file? the pr is empty now 🤔️

**Comment by dombyte** [comment]
Sorry, I'm new to Contributing to git, when revoking the other changed file is must deleted somehow the lang file.

**Comment by floyd-li** [comment]
thanks for your contribution!

**Pull request merged**